packages/blocks/src/page-block/edgeless/components/toolbar/brush/brush-tool-button.ts

```diff
@@ -125,6 +125,14 @@ export class EdgelessBrushToolButton extends WithDisposable(LitElement) {
     super.disconnectedCallback();
   }
 
+  override firstUpdated() {
+    this._disposables.addFromEvent(document, 'keydown', event => {
```

Comment:
Listening events from document may be overkill, which doesn't work well handling multiple editor instanes. @Saul-Mirone do we have API example for subscribing events within editor scope?

Reply:
Yep, we have the `bindHotKey` method on block element.

Reply:
I modified~